cal/cal/templates/tag_detail.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1 class="well">Tag "{{ tag.label }}"</h1>
```

Comment:
A few ways you can tell that it's a Django template:
- It's not in `cal/static/templates/`, just `cal/templates`. Angular templates are served as static files
- There are no verbatim tags before/after the double curly braces.